.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,87 @@
+name: "\U0001F41B Bug Report"
+description: Something isn't working as expected
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Please answer these questions before submitting your issue. Thanks!
+  - type: textarea
+    id: problem
+    attributes:
+      label: What did you do?
+      description: If possible, provide a recipe for reproducing the error.
+    validations:
+      required: true
+  - type: textarea
+    id: expect
+    attributes:
+      label: What did you expect to see?
+    validations:
+      required: false
+  - type: textarea
+    id: see
+    attributes:
+      label: What did you see instead?
+    validations:
+      required: true
```

Comment:
I thing we should only make `What did you do?` section as required and leave all the following optional.

Reply:
Thanks for your comment. The type of this issue is "bug", so I think the other sections also should be required. For `What did you expect to see?` section, I have no idea.